.github/workflows/test.yml

```
@@ -0,0 +1,313 @@
+          cache: 'pnpm'
+
+      - name: 📦 Install dependencies
+        run: pnpm install --frozen-lockfile
+
+      - name: 🔒 Run security audit
+        run: pnpm audit --audit-level moderate
+
+      - name: 🔍 Run CodeQL analysis
+        uses: github/codeql-action/analyze@v3
+        with:
+          languages: typescript
```

actionlint (1.7.7)

input "languages" is not defined in action "github/codeql-action/analyze@v3". available inputs are "add-snippets", "category", "check_name", "checkout_path", "cleanup-level", "expect-error", "matrix", "output", "ram", "ref", "sha", "skip-queries", "threads", "token", "upload", "upload-database", "wait-for-processing"

(action)